bower.json

```diff
@@ -1,21 +1,12 @@
 {
-  "name": "TournamentMango",
+  "name": "tournamentmango",
   "version": "0.7.0",
   "authors": [
     "Kyle Kemp <kyle@seiyria.com>"
   ],
   "dependencies": {
-    "angular-material": "1.0.0",
-    "angular-sortable-view": "0.0.13",
-    "clipboard": "1.4.2",
-    "FileSaver.js": "https://github.com/eligrey/FileSaver.js.git",
-    "firebase": "3",
-    "jquery": "2.1.4",
-    "jspdf": "1.1.135",
     "lodash": "3.10.1",
```

Comment:
I would have removed all the bower dependencies but lodash and tournament-bower are a bit more complicated to move to npm and I didn;t really have time for it. I assume it's just a step in the gulpfile but I'm not experienced with writing gulpfiles. It was only the angular dep that was the issue I believe.